cf_remote/ssh.py

```diff
@@ -46,6 +46,8 @@ def put(self, src, hide=False):
 class Connection:
     def __init__(self, host, user, connect_kwargs=None):
         self.ssh_host = host
+        if ":" in host:
```

Comment:
I don't like this being solved here, TBH. A `Connection` object should get the port as an argument.

Reply:
agreed, will refactor.